doc/contribute_models.md

```diff
@@ -97,7 +89,54 @@ class MyAwesomeClassifier(tf.keras.Model):
         # do custom training here, parameter "dataset" is a tf.dataset type representing the input data.
 ```
 
-In `__init__.py` you should expose your model classes by adding lines like `from mydnnclassifier import MyAwesomeClassifier`. After this, you can test your model following the below step.
+### Export Model Definition
+
+In `__init__.py` you should expose your model classes by adding lines like:
+
+```python
+from awesome_model import MyAwesomeClassifier
+```
+
+### Write the Dockerfile
+
+To build a Docker image, we need a file named `Dockerfile` like below:
```

Comment:
Again, here we need to explain the points -- Please be aware that the base image must be `sqlflow/model_zoo:base`, which contains the SQLFlow server and other necessary stuff.

BTW, what is the difference between `sqlflow/mode_zoo:base` and `sqlflow/sqlflow`? Do we really need the `sqlflow/model_zoo:base`?

Reply:
Done, use `sqlflow/sqlflow` here.